src/index.css

```diff
@@ -1 +1 @@
animation-delay:7500ms !important}.delay-7600{animation-delay:7600ms !important}.delay-7700{animation-delay:7700ms !important}.delay-7800{animation-delay:7800ms !important}.delay-7900{animation-delay:7900ms !important}.delay-8000{animation-delay:8000ms !important}.delay-8100{animation-delay:8100ms !important}.delay-8200{animation-delay:8200ms !important}.delay-8300{animation-delay:8300ms !important}.delay-8400{animation-delay:8400ms !important}.delay-8500{animation-delay:8500ms !important}.delay-8600{animation-delay:8600ms !important}.delay-8700{animation-delay:8700ms !important}.delay-8800{animation-delay:8800ms !important}.delay-8900{animation-delay:8900ms !important}.delay-9000{animation-delay:9000ms !important}.delay-9100{animation-delay:9100ms !important}.delay-9200{animation-delay:9200ms !important}.delay-9300{animation-delay:9300ms !important}.delay-9400{animation-delay:9400ms !important}.delay-9500{animation-delay:9500ms !important}.delay-9600{animation-delay:9600ms !important}.delay-9700{animation-delay:9700ms !important}.delay-9800{animation-delay:9800ms !important}.delay-9900{animation-delay:9900ms !important}.delay-10000{animation-delay:10000ms !important}@keyframes pulse-green{0%{-webkit-box-shadow:0 0 0 0 rgba(48,196,116,.5);-moz-box-shadow:0 0 0 0 rgba(48,196,116,.5);-ms-box-shadow:0 0 0 0 rgba(48,196,116,.5);-o-box-shadow:0 0 0 0 rgba(48,196,116,.5);box-shadow:0 0 0 0 rgba(48,196,116,.5)}70%{-webkit-box-shadow:0 0 0 20px transparent;-moz-box-shadow:0 0 0 20px transparent;-ms-box-shadow:0 0 0 20px transparent;-o-box-shadow:0 0 0 20px transparent;box-shadow:0 0 0 20px transparent}100%{-webkit-box-shadow:0 0 0 0 transparent;-moz-box-shadow:0 0 0 0 transparent;-ms-box-shadow:0 0 0 0 transparent;-o-box-shadow:0 0 0 0 transparent;box-shadow:0 0 0 0 transparent}}@keyframes pulse-blue{0%{-webkit-box-shadow:0 0 0 0 rgba(0,28,88,.5);-moz-box-shadow:0 0 0 0 rgba(0,28,88,.5);-ms-box-shadow:0 0 0 0 rgba(0,28,88,.5);-o-box-shadow:0 0 0 0 rgba(0,28,88,.5);box-shadow:0 0 0 0 rgba(0,28,88,.5)}70%{-webkit-box-shadow:0 0 0 20px transparent;-moz-box-shadow:0 0 0 20px transparent;-ms-box-shadow:0 0 0 20px transparent;-o-box-shadow:0 0 0 20px transparent;box-shadow:0 0 0 20px transparent}100%{-webkit-box-shadow:0 0 0 0 transparent;-moz-box-shadow:0 0 0 0 transparent;-ms-box-shadow:0 0 0 0 transparent;-o-box-shadow:0 0 0 0 transparent;box-shadow:0 0 0 0 transparent}}.pulse{animation-name:pulse;animation-delay:0s;animation-timing-function:ease;animation-play-state:running}@keyframes smoke-1{0%{transform:translate(0px, 50px);opacity:1}80%{opacity:0}100%{transform:translate(0px, 0px);opacity:0}}@keyframes smoke-2{0%{transform:translate(0, 50px);opacity:1}80%{opacity:0}100%{transform:translate(0, 0);opacity:0}}@keyframes smoke-3{0%{opacity:1;transform:translate(0px, 80px)}80%{opacity:0}100%{transform:translate(0, 0);opacity:0}}.ck-editor{font-family:"Brandon Grotesque Regular"}.ck-focused{border-color:#c4c4c4 !important}/*# sourceMappingURL=index.css.map */
```

Comment:
@william-gooch please remove changes to this file it course conflict and is generation automatically just leave this to @farmanahmed2007